### DISCUSSION TITLE
Concept option for help command card-83944797

### DISCUSSION THREAD

**Discussion by thatrandomperson5** [JDJGInc/JDBot#74] [Ideas]
The card: [here](https://github.com/JDJGInc/JDBot/projects/1#card-83944797)
The concept: [here](https://github.com/thatrandomperson5/JDBot/blob/concepts/cogs/help.py)
The [lines](https://github.com/thatrandomperson5/JDBot/blob/83e24439f07fe44c4698ab8b42229aab8451a34a/cogs/help.py#L111) that I added.

# Idea: Make help command cog names case-insensitive
## pros
Will make the help command easier to use.
Requires no external lib.
Reduces confusion.
## cons
Some commands will need to be renamed.
Adds a ugly chunk of code to the help command cog

**Comment by thatrandomperson5** [comment]
Do you think this a good ideas? 👍 /👎

**Reply by JDJGInc** [reply]
I would rather have an explanation in help or some quick help guide that appears that the users can use when using the bot.

**Reply by thatrandomperson5** [reply]
Ok

**Reply by JDJGInc** [reply]
and maybe making the help command pop more cause it's a bit bland rn.

**Reply by thatrandomperson5** [reply]
I’m not really a stylist

**Comment by thatrandomperson5** [comment]
This discussion is ended I think. Idk how to end it so I’m saying it here.
